benchmarking/mod/benchmark.py

```diff
@@ -152,6 +152,12 @@ def run(self):
         dest_dir = self.cluster["tmp_dir"]
         nodes = self.cluster["osd"]
         monitor_interval = self.cluster["monitoring_interval"]
+
+        #clean up worker environment
+        worker = self.cluster["osd"] + self.cluster["client"]
+        common.pdsh(user,nodes,"rm -rf {0}/*".format(self.cluster["tmp_dir"]))
```

Comment:
No, I suggest you do not do such things in codes, remove all is definitely the mustn't do in codes. Please just remove the file created by CeTune